tools/cmake/common/preset.cmake

```diff
@@ -89,11 +89,9 @@ endmacro()
 # Set an overridable option.
 macro(set_overridable_option NAME VALUE)
   # If the user has explitily set the option, do not override it.
-  if(DEFINED ${NAME})
-    return()
```

Comment:
`return` not exiting the macro?

Reply:
Return was exiting the scope that `set_overridable_option` was used in since it's a macro. We now just avoid that overall